src/install/class-wordlift-install-3-50-1.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Wordlift\Jsonld\Jsonld_Utils;
+
+/**
+ * @since 3.49.1
+ */
+class Wordlift_Install_3_50_1 extends Wordlift_Install {
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected static $version = '3.50.1';
+
+	/**
+	 * Is column exists
+	 *
+	 * @param $column_name
+	 *
+	 * @return mixed
+	 */
+	public static function is_column_exists( $column_name ) {
+		global $wpdb;
+
+		return $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name ='{$wpdb->prefix}wl_entities' AND column_name = %s",
+				$column_name
+			)
+		);
+	}
+
+	/**
+	 * Install
+	 *
+	 * @return void
+	 */
+	public function install() {
+		global $wpdb;
+
+		// Check if 'match_name' column exists
+		if ( self::is_column_exists( 'match_name' ) ) {
```

Comment:
Do you need this when using dbDelta?

Reply:
Sure, if you're using `dbDelta`, you technically don't need the extra check for column existence. But if you're dealing with a large database, skipping the check could slow things down. So in that case, the check could still be useful for performance reasons.